Resources/Storage/XROOTStorage.py

```diff
@@ -1597,3 +1598,32 @@ def getCurrentURL( self, fileName ):
     cwdUrl = result['Value']
     fullUrl = '%s/%s' % ( cwdUrl, fileName )
     return S_OK( fullUrl )
+
+  def constructURLFromLFN( self, lfn, withWSUrl = False ):
```

Comment:
That is an excelent question ! :-) 
I can just tell you that we (LHCb) do run these tests...